daemon/api_sideload_n_try.go

```diff
@@ -32,29 +34,72 @@ import (
 	"github.com/snapcore/snapd/client"
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/logger"
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/overlord/assertstate"
-	"github.com/snapcore/snapd/overlord/auth"
 	"github.com/snapcore/snapd/overlord/snapstate"
 	"github.com/snapcore/snapd/overlord/state"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/snapfile"
 )
 
-const maxReadBuflen = 1024 * 1024
+type Form struct {
+	Value      map[string][]string
+	FileHeader map[string][]*FileHeader
+}
+
+type FileHeader struct {
```

Comment:
nitpick^2: "file header" can be a bit ambiguous (I would expect this to be the first few bytes from the file); maybe "FileReference"?

Reply:
Renamed that and also changed the fields to be plural since there can be many file/values per name. Thanks